nemo/collections/asr/parts/utils/diarization_utils.py

```diff
@@ -237,7 +237,10 @@ def run_diarization(
         diar_model = ClusteringDiarizer(cfg=diar_model_config)
         score = diar_model.diarize()
         if diar_model_config.diarizer.vad.model_path is not None and not diar_model_config.diarizer.oracle_vad:
-            self.get_frame_level_VAD(vad_processing_dir=diar_model.vad_pred_dir)
+            self.get_frame_level_VAD(
+                vad_processing_dir=diar_model.vad_pred_dir,
+                smoothing_type=diar_model_config.diarizer.vad.parameters.smoothing,
```

Comment:
I'm worried the smoothing does not present in  diar_model_config.diarizer.vad and looks like we do not have a check/default value in diar_model.
Approving this but I think we should have general default values for diar_model_config to avoid dict access error.  @tango4j